src/components/SearchResults/UserInfo/UserInfo.js

```diff
@@ -1,17 +1,29 @@
 import styles from "./UserInfo.module.css";
 
-const UserInfo = ({ userInfo: { image, name, handle, bio, date } }) => {
+const UserInfo = ({
+  userInfo: { avatar_url, name, login, bio, created_at },
+}) => {
   return (
     <div className={styles.userInfo}>
-      <img className={styles.userImage} src={image} alt="User avatar" />
+      <img className={styles.userImage} src={avatar_url} alt="User avatar" />
       <div className={styles.userInfoIntro}>
         <div className={styles.userInfoName}>
-          <h2>{name}</h2>
-          <p className={styles.userInfoHandle}>{handle}</p>
+          <h2>{`${name || "Not Available"}`}</h2>
+          <p className={styles.userInfoHandle}>{`${
+            login || "Not Available"
+          }`}</p>
         </div>
-        <p className={styles.userInfoDate}>{date}</p>
+        <p className={styles.userInfoDate}>{`Joined ${
+          new Date(created_at).toLocaleString("en-US", {
```

Comment:
1. You can take this logic out of this section and have it at the top of the file. 
2. With the current implementation, when there is no date, it returns the string "Invalid Date" so "Not Available" will never be reached. A way to handle this would be to check whether the date itself exists, then choose what to render.

Here is how I did it:

```suggestion
          const formattedDate = created_at
    ? new Date(created_at).toLocaleString("en-US", {
        day: "2-digit",
        month: "short",
        year: "numeric",
      })
    : "YY/MM/DD";
```